src/Outsystems.SetupTools/Lib/PlatformSetup.ps1

```diff
@@ -66,9 +86,16 @@ function GetDotNet4Version()
 {
     LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 2 -Message "Getting the registry value HKLM:SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\<langid>\Release."
 
+    <#
+        RPD-4212: For Windows installations with Japanese language, registry has two entries located at
```

Comment:
Should we commit to the public repo the internal issue number? I don't actually see any problem with this, just making a note.